yt_dlp/extractor/franceculture.py

```diff
@@ -1,27 +1,39 @@
 # coding: utf-8
 from __future__ import unicode_literals
 
+import re
 from .common import InfoExtractor
 from ..utils import (
     determine_ext,
     extract_attributes,
     int_or_none,
+    traverse_obj,
 )
 
 
 class FranceCultureIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?franceculture\.fr/emissions/(?:[^/]+/)*(?P<id>[^/?#&]+)'
     _TESTS = [{
-        'url': 'http://www.franceculture.fr/emissions/carnet-nomade/rendez-vous-au-pays-des-geeks',
+        # playlist
+        'url': 'https://www.franceculture.fr/emissions/hasta-dente',
```

Comment:
Add a test for the entries as well. Like this:
```py
        'url': ...,
        'info_dict': {
            ...
        },
        'playlist': [{
            'md5': ...,
            'info_dict': {
                ...
            },
        },
        'params': {'playlist_end': 1},
```

Reply:
Done thanks.